flocx_market/.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URI=mysql+pymysql://flocx_market:qwerty123@127.0.0.1:3307/flocx_market
+FLASK_APP=api.py
```

Comment:
Not entirely sure what this is used for... ? In any case, starting up the API should be done through an entrypoint in setup.cfg - @ljmcgann and @DanNiESh should be able to help with that.

Reply:
The `.env` file is used by `docker-compose`; it should not be part of this commit.